compiler/src/parsing/parser.messages

```diff
@@ -404,7 +404,7 @@ program: MODULE UIDENT EOL FROM UIDENT YIELD
 
 Expected the keyword `use` followed by items to use surrounded by `{}` or `*` to use all bindings.
```

Comment:
@ospencer don't the error messages need to be updated?

Reply:
I thought I had checked all of them, and I did, but I missed just this one. Good catch!